src/lsm/manifest_level.zig

```diff
@@ -546,7 +546,7 @@ pub fn ManifestLevelType(
                 if (optimal == null or range.tables.len < optimal.?.range.tables.len) {
                     optimal = LeastOverlapTable{
                         .table = TableInfoReference{
-                            .table_info = @intToPtr(*TableInfo, @ptrToInt(table)),
```

Comment:
We can replace this pattern `@intToPtr(*X, @ptrToInt(x_const))` with `@constCast` introduced in Zig 0.11.0.